src/components/FeatureCard.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import FeatureButton from './FeatureButton.jsx';
+
+/**
+ * @param {function} event
+ *  @returns {JSX} jsx
+ */
+export default class FeatureCard extends Component {
+  /**
+     * @param {function} render
+     *  @returns {JSX} jsx
+     */
+  render() {
+    const {
+      imgUrl,
+      title,
+      activeClass,
+    } = this.props;
+    return (
+      <div id="cards" className={activeClass} style={
+        {
+          background: `linear-gradient(rgba(0, 0, 0, 0.5), 
+                        rgba(0, 0, 0, 0.5)), url(${imgUrl}) no-repeat center `,
+          backgroundSize: 'cover',
+        }
+      }>
+        <div id="feature_button_div">
+          <h5>{title}</h5>
+          <FeatureButton
+            text="View"
+            myclass={
+              activeClass
+                .localeCompare(
```

Comment:
What are you trying to do here?

Reply:
I am trying to compare two strings, if they are the same, assign a CSS class to the component. If they are different assign a different CSS class. localCompare is method of javascript String class

Reply:
Yea I know it compares strings. But since you already know what you are comparing against, you could as well do `activeClass === 'cards-active'` which is shorter and saves some behind the scene computation. Or is there something else you want to achieve with that?

Reply:
done